assets/cubyz/biomes/savannah.zig.zon

```diff
@@ -0,0 +1,84 @@
+.{
+	.properties = .{
+		.dry,
+		.land,
+	},
+	.minHeight = 30,
+	.maxHeight = 40,
+	.minRadius = 100,
+	.maxRadius = 160,
+	.hills = 6,
+	.validPlayerSpawn = true,
+	.music = "cubyz:EasternThought",
+	
+	.ground_structure = .{
+		"3 to 4 cubyz:limestone",
+	},
+	.structures = .{
+		.{
+			.id = "cubyz:simple_vegetation",
+			.block = "cubyz:cactus",
+			.chance = 0.02,
+			.height = 1,
+			.height_variation = 3,
+		},
+		.{
+			.id = "cubyz:ground_patch",
+			.block = "cubyz:dirt",
+			.chance = 0.33,
+			.width = 5,
+			.variation = 4,
+			.depth = 2,
+			.smoothness = 0.1,
+		},
+		.{
+			.id = "cubyz:ground_patch",
+			.block = "cubyz:dry_grass",
+			.chance = 0.33,
+			.width = 5,
+			.variation = 4,
+			.depth = 2,
+			.smoothness = 0.1,
+		},
+		.{
+			.id = "cubyz:simple_tree",
+			.leaves = "cubyz:baobab_leaves",
+			.log = "cubyz:baobab_log",
+			.top = "cubyz:baobab_top",
+			.chance = 0.075,
+			.type = .round,
+			.height = 1,
+			.height_variation = 7,
+			.leafRadius = 2,
+			.leafRadius_variation = 1,
+		},
+	},
+	.transitionBiomes = .{
+		.{
+			.id = "cubyz:trans/savannah_coast",
+			.chance = 1,
+			.width = 1,
+			.keepOriginalTerrain = 0.5,
+			.properties = .{.ocean},
+		},
+		.{
+			.id = "cubyz:trans/sparse_bushlands",
+			.chance = 1,
+			.width = 2,
+			.keepOriginalTerrain = 1,
+			.properties = .{.neitherWetNorDry, .inland, .land},
+		},
+		.{
+			.id = "cubyz:trans/savannah_widow",
```

Comment:
In my opinion it looks kind of wrong to have a zone without trees there.
Do we really want transition biomes for everything?
![Screenshot at 2025-01-08 23-26-40](https://github.com/user-attachments/assets/de0c9b74-a197-46f3-aaa9-60dc055f1c5a)